cognite_toolkit/cdf.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# The Typer parameters get mixed up if we use the __future__ import annotations
```

Comment:
This is the second time I have done this mistake and spent a lot of time figuring it out, which is why it is now a comment.

Reply:
ohhhh...